### PR TITLE
rmblast update

### PR DESCRIPTION
**Pull request by pvanheus**
* add makeblastdb test
* add bzip2 dependency

* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by tseemann** [comment]
@pvanheus you added `makeblastdb` as a test. that will fail and return 1. Maybe use `makeblastdb -help` ?

**Comment by pvanheus** [comment]
@tseemann the raw `makeblastdb` was just an attempt to see what it was doing - to see its output. The original test was `makeblastdb 2>&1 |grep nucl >/dev/null` which returns 0 as expect in my testing but doesn't work on Travis.

**Comment by pvanheus** [comment]
The problem is now clear - this is building against the bzip2 package from before https://github.com/conda-forge/bzip2-feedstock/pull/1 was merged. Not sure if it is possible to specify that bzip2 1.0.6  build number > 0 is needed. bzip2 >=1.0.6--1 doesn't work.

**Comment by bgruening** [comment]
`1.0.6_1` will work imho.

File: recipes/rmblast/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,26 @@ source:
...
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
-    - boost # [linux]
+    - boost <=1.58 # [linux]
     - gcc # [linux]
+    - bzip2 >=1.0.6 1 # [linux]
```
Comment (nsoranzo) [inline recipes/rmblast/meta.yaml]: If you specify a build number, then the version must be exact:
```
    - bzip2 1.0.6 1 # [linux]
```

**Comment by bgruening** [comment]
Everything looks fine, the correct version of bzip2 is installed. It seem that makeblastdb does not find the shared-object from bzip? @pvanheus do you know if makeblastdb is compiled as well in this PR or is this some kind of static binray that is shipped?

**Comment by pvanheus** [comment]
Closing this one so that I can do a cleaner one.

**Pull request closed without merging**